arch/arm/boot/dts/qcom/batterydata-yukon-flamingo.dtsi

```diff
@@ -72,7 +72,7 @@ qcom,flamingo-batterydata {
 	};
 
 	qcom,rbatt-sf-lut {
-		qcom,lut-col-legend = <(-20) 0 25 40 60 70>;
```

Comment:
This looks REALLY wrong. You are lowering the algorithm precision by removing that extra value.